javalib/src/main/scala/java/util/StringTokenizer.scala

```diff
@@ -1,103 +1,93 @@
 package java.util
 
-class StringTokenizer(string: String,
-                      private var delimiters: String,
-                      returnDelimiters: Boolean)
-    extends java.util.Enumeration[Object] {
+class StringTokenizer(
+  str: String,
+  private var delim: String,
+  returnDelims: Boolean
+) extends java.util.Enumeration[Object] {
 
-  def this(string: String) = this(string, " \t\n\r\f", false)
-  def this(string: String, delimeters: String) =
-    this(string, delimeters, false)
+  def this(str: String) = this(str, " \t\n\r\f", false)
+  def this(str: String, delim: String) = this(str, delim, false)
 
-  if (string == null) {
-    throw new NullPointerException()
-  }
-
-  private var position = 0
+  private var position: Int = 0
```

Comment:
At the risk of passing on cargo cult programming & superstition, somewhere I was taught that the preferred style in Scala Native was to not give the type after val/var
declarations unless they added some benefit.  This is the minimalist Haskell style, rather than the fully declarative Pascal style.
Did someone in Scala.js land recommend giving the type as a matter of course?
Perhaps my novice training was wrong or has been superseded.  I think declaring the type here makes no functional change.